ls1intum/Artemis
# `Development`: Improve file name input validation

### Checklist
#### General


#### Server

### Motivation and Context


The input validation

### Description
The input validation is now applied in more instances and checks for incorrect multiple sequential dots.

### Steps for Testing
We just want to make sure that I didn't accidentally break the implementation.

Prerequisites:
- 1 Instructor

Split Lecture PDF (Instructor):
1. Open the lecture tab and click on "Create Lecture"
4. Fill in the title and channel name
5. Check "Automatic unit processing" at the bottom
7. Select a multipage PDF to process and hit "Process units". If you use a PDF that fits the requirements, skip point 5. Otherwise, continue.
8. Add units manually, at least two and set the pages accordingly. Then hit "submit.
9. Open the lecture and download a lecture unit attachment.
11. No issues should occur

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage
